canonical-web-and-design/buy.ubuntu.com
# add js function to allow updating of quantity input on cart page

## Done

- put existing minified js through an "unminifier", and added the file to the project
- added uglify-js
- removed `readonly` attribute from quantity input on cart page
- added function (on line #3040 of "canonical.js") to unminifed js to allow line item quantities to be updated via the relevant input field on the cart page

## QA
- Visit https://canonical-staging.myshopify.com/
- Add any product to your cart
- On the cart page, type a new quantity into the input field, and hit return
- See that the quantity and totals have been updated appropriately
